### PR TITLE
feat(deploy): deploy github pages

### PR DESCRIPTION
**Pull request by mehdirajani5**
Feature deploy GitHub pages.
Fixed issue  #7

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Deploy demo
+        uses: AhsanAyaz/angular-deploy-gh-pages-actions@v1.3.1
+        with:
+          github_access_token: ${{ secrets.ACCESS_TOKEN }}
+          build_configuration: production
+          base_href: /ngx-read-time-ws/
```
Comment (AhsanAyaz) [inline .github/workflows/main.yml]:
```suggestion
          base_href: /ngx-read-time/
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+          deploy_branch: gh-pages
+          angular_dist_build_folder: projects/demo/dist/demo
```
Comment (AhsanAyaz) [inline .github/workflows/main.yml:19]: Are you 100% sure `projects/demo/dist/demo` is the build directory?

**Review by AhsanAyaz — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 2 above)
Comment (mehdirajani5) [reply .github/workflows/main.yml:19]: As far as I know yhi link bn rha tha.

Comment (mehdirajani5) [reply .github/workflows/main.yml:19]: Well merge it, If it causes any issue we will look into it together.

**Pull request merged**